src/OrchardCore/OrchardCore.DisplayManagement/Shapes/CoreShapes.cs

```diff
@@ -104,10 +104,15 @@ public IHtmlContent Message(dynamic Shape)
             TagBuilder tagBuilder = OrchardCore.DisplayManagement.Shapes.Shape.GetTagBuilder(Shape, "div");
             string type = Shape.Type.ToString().ToLowerInvariant();
             IHtmlContent message = Shape.Message;
-            tagBuilder.AddCssClass("message");
-            tagBuilder.AddCssClass("message-" + type);
+            tagBuilder.AddCssClass("alert");
+            tagBuilder.AddCssClass("alert-" + type);
+            tagBuilder.AddCssClass("alert-dismissible fade show");
             tagBuilder.Attributes["role"] = "alert";
```

Comment:
I like the idea, but not the implementation sorry

@agriffard wanted to do exactly the same thing on https://github.com/OrchardCMS/OrchardCore/pull/4538 but decided to implement our own css to do so and keep the `message` class

See https://github.com/OrchardCMS/OrchardCore/pull/4579 for how it was done. 

Suggest you extend that in same way, as @ns8482e is right, this shape is designed to be used by themes that are not implementing bootstrap

Reply:
This is how it was before, if you like the shape style we need to extract the entire markup

Reply:
@hishamco if you have a look in `TheAdmin.scss` file you'll find the messages section where bootstrap has already been extended.

```
// Messages
// ------------------------------
.message {
    @extend .alert;
}

.message-success {
    @extend .alert-success;
}

.message-information {
    @extend .alert-info;
}

.message-warning {
    @extend .alert-warning;
}

.message-error {
    @extend .alert-danger;
}
```

you should just be able to add `@extend .alert-dismissable` and the fade / show mixins to the sass without changing any of the Shape classes, or having to extract the entire markup

Reply:
I got what you mean, let me try

Reply:
I added `@extend .alert-dismissable` but the close button doesn't work as expected